src/classes/MAPR_RELAutoCreateSettings.cls

```diff
@@ -0,0 +1,123 @@
+    public virtual List<MODL_ConRelationshipAutoCreateMapping> getContactMappingModelsFromSettings(){
+        if(MAPR_RELAutoCreateSettings.relationshipAutocreateMappingsCache != null) {
+            return MAPR_RELAutoCreateSettings.relationshipAutocreateMappingsCache;
+        }
+
+        Map<Schema.SObjectType, List<Relationship_Auto_Create__c>> relAutoCreateListBySObjectType =
+            UTIL_CustomSettingsFacade.getAutoCreateSettingsForContact();
+        MAPR_RELAutoCreateSettings.relationshipAutocreateMappingsCache =
+            this.getContactMappingModelsFromMappings(relAutoCreateListBySObjectType.get(Contact.SObjectType));
+
+        return MAPR_RELAutoCreateSettings.relationshipAutocreateMappingsCache;
+    }
+
+    /***************************************************************************************************************************
+    * @description Generates a list of MODL_ConRelationshipAutoCreateMapping representing a specified list of Relationship
+    * Auto Create mappings.
+    * @param relAutoCreateList List of Relationship Auto Create settings to be used to generate mapping models.
+    * @return A list of MODL_ConRelationshipAutoCreateMapping representing the Relationship Auto Create mappings provided.
+    ***************************************************************************************************************************/ 
+    @TestVisible
+    protected  virtual List<MODL_ConRelationshipAutoCreateMapping> getContactMappingModelsFromMappings(List<Relationship_Auto_Create__c> relAutoCreateList){
+        List<MODL_ConRelationshipAutoCreateMapping> conRelAutoCreateMapModelsList = new List<MODL_ConRelationshipAutoCreateMapping>();
+
+        for (Relationship_Auto_Create__c relAutoCreate : relAutoCreateList){
+            MODL_ConRelationshipAutoCreateMapping relatedModel = this.getModelFromMapping(relAutoCreate);
+
+            if (relatedModel != null){
+                conRelAutoCreateMapModelsList.add(relatedModel);
+            }
+        }
+
+        return conRelAutoCreateMapModelsList;
+    }
+
+    /***************************************************************************************************************************
+    * @description Generates a single of MODL_ConRelationshipAutoCreateMapping representing a specified Relationship
+    * Auto Create mapping.
+    * @param relAutoCreate A Relationship Auto Create mapping from which to generate a model.
+    * @return A MODL_ConRelationshipAutoCreateMapping representing the specified Relationship Auto Create mapping.
+    ***************************************************************************************************************************/
+        @TestVisible
+    protected virtual MODL_ConRelationshipAutoCreateMapping getModelFromMapping(Relationship_Auto_Create__c relAutoCreate){
+        Schema.DescribeFieldResult fieldDescribe = UTIL_Describe.getFieldDescribe(relAutoCreate.Object__c, relAutoCreate.Field__c);
```

Comment:
What is the value in getting fieldDescribe vs. just sticking with the API name of the field as it worked before? I haven't run any CPU metrics on this but I'm guessing that getting the Describe adds to the CPU total.

This method is actually introducing a functionality change. If it can't find the field as configured, it returns null and skips the auto create setting entirely when evaluating contacts (vs. throwing an error on the contact insert/update). I'm not saying the error on every contact change is desirable, but it's a change.